scripts/hypothesistooling.py

```diff
@@ -387,6 +389,8 @@ def should_run_ci_task(task, is_pull_request):
     # know are safe to ignore, and run tests if there's anything left.
     changed_files = changed_files_from_master()
 
+    print('@@AWLC changed_files = %r' % changed_files)
```

Comment:
Looks like a debugging print - remove this and the rest LGTM.

Reply:
I will, although the code only started working when I put it in! 😭